README.md

```diff
@@ -219,3 +219,6 @@ MaxText supports training and inference of various open models. Follow user guid
 
 * [Gemma](https://ai.google.dev/gemma): a family of open-weights Large Language Model (LLM) by [Google DeepMind](https://deepmind.google/), based on Gemini research and technology. \
 You can run decode and finetuning using instructions mentioned [here](https://github.com/google/maxtext/blob/main/getting_started/Run_Gemma.md).
+
+## Automatically Upload Logs to Vertex Tensorboard
+MaxText supports automatic upload of logs collected in a directory to a Tensorboard instance in Vertex AI. Follow [user guide](https://github.com/google/maxtext/tree/main/getting_started/Use_Vertex_AI_Tensorboard.md) to know more.
```

Comment:
nit: should be `[user guide](getting_started/Use_Vertex_AI_Tensorboard.md)` or similar. (That way when folks make a change their thing updates. (I'm trying to move other absolute links to be relative links.)